opennms-webapp/src/main/webapp/report/index.jsp

```diff
@@ -149,6 +149,8 @@
 </div>
   <hr />
 
-<jsp:include page="/js/angular.min.jsp" flush="false" />
-<jsp:include page="/js/search.min.jsp" flush="false" />
+<jsp:include page="/assets/load-assets.jsp" flush="false">
```

Comment:
Use a valid `.jsp`. The other `.jsp`s included here don't actually exist. This caused to fail the smoke tests that used to page to fail *sometimes*.